openedx_ledger/models.py

```diff
@@ -27,6 +28,7 @@ class UnitChoices:
         (SEATS, 'Seats in a course'),
         (JPY, 'Japanese Yen'),
     )
+    history = HistoricalRecords()
```

Comment:
This class is just a plain old python class, not a Django model, so it can't use simple history.